packages/algoliasearch-helper/src/algoliasearch.helper.js

```diff
@@ -1614,6 +1614,13 @@ AlgoliaSearchHelper.prototype._recommend = function () {
       })
     );
 
+    ids = Array.prototype.concat.apply(
+      ids,
+      derivedState.params.map(function (param) {
+        return param.$$id;
+      })
+    );
```

Comment:
This shouldn't be in this PR, should it?

Reply:
This is probably an unwanted merge/rebase resolution, as it duplicates the statement before. It can safely be removed.